ipaserver/install/server/__init__.py

```diff
@@ -177,6 +176,9 @@ class ServerInstallInterface(ServerCertificateInstallInterface,
     preserve_sssd = False
     no_sssd = False
 
+    force_join = client.ClientInstallInterface.force_join
+    force_join = replica_install_only(force_join)
```

Comment:
This bit seems to be unnecessary, since you are masking `force_join` in `ServerMasterInstall` with a constant.